.github/workflows/swift-toolchain.yml

```diff
@@ -245,11 +250,20 @@ jobs:
           show-progress: false
 
       - uses: compnerd/gha-setup-vsdevenv@main
+        if: inputs.builder_os == 'Windows'
```

Comment:
I think that this is a no-op on other targets, so we can safely just run it always.